eq-author/src/App/page/Design/QuestionPageEditor/AnswersEditor/AnswerEditor/index.js

```diff
@@ -115,15 +114,6 @@ class AnswerEditor extends React.Component {
   }
 
   getAnswerTypeText(answer) {
-    if (answer.type === UNIT) {
-      const unitConfig = unitConversion[answer.properties.unit];
-      return (
-        <>
-          {unitConfig.unit}
-          {` (${unitConfig.abbreviation})`}
-        </>
-      );
-    }
```

Comment:
You can't remove this block of code because it adds the title to the top of answer. There needs to be a placeholder for when no unit is selected and that needs to be swapped out when the user does select a unit